Explorer/Assets/DCL/SDKComponents/Tween/Components/SDKTweenComponent.cs

```diff
@@ -0,0 +1,22 @@
+using Arch.Core;
+using DCL.ECSComponents;
+using DG.Tweening;
+using UnityEngine;
+
+namespace ECS.Unity.Tween.Components
+{
+    public struct SDKTweenComponent
+    {
+        public Entity globalWorldEntity;
```

Comment:
Your `globalWorldEntity` can just be a regular Entity from the system's `World` (the scene-level world).

Reply:
I was leaving it around in case I needed it (as I copied this class from old renderer) but it was not even used :) good catch